understanding/language-of-page.html

```diff
@@ -141,51 +125,46 @@ <h2>Intent</h2>
                </div>
             </section>
             <section id="benefits">
-               <h2>Benefits</h2>
-               <p>This Success Criterion helps:</p>
+               <h2>利点</h2>
+               <p>この達成基準は、次のような利用者にメリットがある:</p>
                <ul>
                   
                   
-                  <li>people who use screen readers or other technologies that convert text into synthetic
-                     speech;
+                  <li>スクリーンリーダー又はテキストを合成音声に変換するその他の技術を使用している利用者。
                      
                   </li>
                   
                   
-                  <li>people who find it difficult to read written material with fluency and accuracy, such
-                     as recognizing characters and alphabets or decoding words;
+                  <li>例えば、文字及びアルファベットを認識したり、単語を読み取ったりすることのように、流暢かつ正確に書かれたものを読むことが困難な利用者。
```

Comment:
どうにも変なような。カンマ前後を原文の順に戻しつつ、文末に「困難」を補うとこんな感じですが、これでどうでしょうか。

```suggestion
                  <li>流暢かつ正確に書かれたものを読むことが困難な利用者、例えば、文字及びアルファベットを認識したり、単語を読み取ることなどが困難。
```

Reply:
他の段落が「利用者」で終わってる文なので文末は揃えたいですね…と思ったら、peopleを利用者としてしまってるんですね…ということで「利点」セクションをまるっと修正してみました。